doc/langdef.md

```diff
@@ -698,6 +698,26 @@ notation like `_+_` where `_` is a placeholder for an argument.
 TODO https://issuetracker.google.com/67014381 : have better descriptions. The
 table is auto-generated so the descriptions need to be updated in the code.
 
+### Timezones
+
+Timezones are expressed in the following grammar:
+```grammar
+TimeZone = "UTC" | LongTZ | FixedTZ ;
+LongTZ = ? list available at http://joda-time.sourceforge.net/timezones.html ? ;
+FixedTZ = ( "+" | "-" ) Digit Digit ":" Digit Digit ;
+Digit = "0" | "1" | ... | "9" ;
+```
+
+Fixed timezones are explicit hour and minute offsets from UTC.  Long timezone
```

Comment:
Was this set of formats double checked against what the other evaluators support? I'm thinking mostly of C++

Reply:
This was the intersection of what the Java and C++ implementations supported, as could be inferred by reading the code.